packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -17,9 +18,16 @@ const NominationBanner = ({ nomination }) => {
   const minutes = newDate.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
   const finalDate = `${time} – ${minutes}`;
   const [activeNomination, setActiveNomination] = useContext(ActiveNominationContext);
-  const declineStatus = 'Declined'
+  
+
```

Comment:
Nitpick: A single blank line is probably OK, especially mid-function. There are also a few cases of extra blank lines in the CSS file. No big deal, but over time these things diminish readability.

Reply:
Done!